samples/Nancy.Demo.Authentication.Forms.TestingDemo/TestRootPathProvider.cs

```diff
@@ -0,0 +1,26 @@
+namespace Nancy.Demo.Authentication.Forms.TestingDemo
+{
+    using System;
+    using System.IO;
+    using Nancy.Testing;
+
+    public class TestRootPathProvider : IRootPathProvider
+    {
+        public string GetRootPath()
+        {
+            var assemblyFilePath =
+                new Uri(typeof(FormsAuthBootstrapper).Assembly.CodeBase).LocalPath;
+
+            var assemblyPath =
+                Path.GetDirectoryName(assemblyFilePath);
+
+            var rootPath =
+                PathHelper.GetParent(assemblyPath, 3);

```

Comment:
@khellang :stuck_out_tongue: it's the code from the old `FakeRootPathProvider`, just copied it over to kill the class. I think we should retire the `Nancy.Demo.Authentication.Forms.TestingDemo` sample and move it over to the Forms Authentication section of the wiki. The demo basically just shows how you can unit tests forms authentication code